cartoframes/data/dataset.py

```diff
@@ -578,7 +578,8 @@ def _rows(self, df, cols, with_lnglat, geom_col):
                     csv_row += '{val}|'.format(val=val)
 
             if the_geom_val is not None:
-                geom = decode_geometry(the_geom_val)
+                enc_type = detect_encoding_type(the_geom_val)
+                geom = decode_geometry(the_geom_val, enc_type)
                 if geom:
                     csv_row += 'SRID=4326;{geom}'.format(geom=geom.wkt)
```

Comment:
Could we try if this works:

```suggestion
                    csv_row += '{geom}'.format(geom=geom.wkb)
```

we might save some bytes in the request payload for each row.

Reply:
This is not working because it puts directly the bytes:

```

Reply:
This is not working because it puts directly the bytes

```
0|0|b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00H\x93@\x00\x00\x00\x00\x00\x9d\xb6@'
10|10|b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00H\x93@\x00\x00\x00\x00\x00\x9d\xb6@'
20|20|b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00H\x93@\x00\x00\x00\x00\x00\x9d\xb6@'
30|30|b'\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00H\x93@\x00\x00\x00\x00\x00\x9d\xb6@'
```